sqlglot/dialects/tsql.py

```diff
@@ -224,6 +217,19 @@ def _format_sql(self: TSQL.Generator, expression: exp.NumberToStr | exp.TimeToSt
     return self.func("FORMAT", expression.this, fmt_sql, expression.args.get("culture"))
 
 
+def _build_datepart(self: TSQL.Generator, expression: exp.Extract) -> str:
+    DATE_PART_UNMAPPING = {
+        "WEEKISO": "ISO_WEEK",
+        "DAYOFWEEK": "WEEKDAY",
+        "TIMEZONE_MINUTE": "TZOFFSET",
+    }
```

Comment:
Why do we need to unmap? Can we just have the correct representation at parse time? Also, when defining constants like this one, let's bubble them up to the global scope.